content/blog/2020/10/2020-10-28-election-candidates.adoc

```diff
@@ -0,0 +1,457 @@
+  This blog post announces candidates for the Jenkins Governance Board and Officer roles.
+  Voting will start on Nov 10, voter registration is open until Nov 08.
+  Results will be announced on Dec 03.
+opengraph:
+  image: /images/governance/elections/2020/opengraph.png
+---
+
+:toc:
+
+As you probably know, in a few weeks we will have the Jenkins 2020 elections.
+We will be electing two link:/project/board[governance board] and for all five link:/project/team-leads/[officer] positions,
+namely: Security, Events, Release, Infrastructure, and Documentation.
+After the link:/blog/2020/09/24/board-elections/[announcement on Sep 24],
+we have been accepting nominations from community members.
+
+After the processing and confirmations with potential candidates,
+the Jenkins 2020 Elections committee is happy to announce the candidates for the Jenkins Governance Board and Officer roles:
+
+* Governance Board candidates: Andrey Falko, Ewelina Wilkosz, Frederic Gurr, Gavin Mogan, Justin Harringa, Mark Waite, Marky Jackson, Steven Terrana, Zhao Xiaojie (Rick)
+* Release officer: Baptiste Mathus, Tim Jacomb, Victor Martinez
+* Security officer: Daniel Beck (uncontested)
+* Events officer: Marky Jackson (uncontested)
+* Infrastructure Officer: Olivier Vernin (uncontested)
+* Documentation officer: Mark Waite (uncontested)
+
+We encourage all community members to support the candidates and to participate in the elections!
+
+== Key dates
+
+* **Nov 08** - Voting sign-up is over.
+* **Nov 10** - Voting begins. link:https://civs.cs.cornell.edu/[Condorcet Internet Voting Service] will be used for voting.
+* **Nov 27** - Voting ends, 11PM UTC.
+* **Dec 03** - Election results are announced and take effect.
+
+== Signing up for voting
+
+Any Jenkins individual contributor is eligible to vote in the election
+if there was a contribution made before September 01, 2020.
+_Contribution_ does not mean a _code contribution_,
+all contributions count:
+documentation patches,
+code reviews,
+substantial issue reports,
+issues and mailing list responses,
+social media posts,
+testing,
+etc.
+Such a contribution should be public.
+
+You can register to vote in one of two ways:
+
+1. Fill out link:https://forms.gle/y3qDo8EM8iQnd3fY9[this Google Form].
+   This way requires logging into your Google Account to verify authenticity.
+2. Send an email to mailto:jenkins-2020-elections@googlegroups.com[jenkins-2020-elections@googlegroups.com].
+   You will need to provide the information specified link:/project/board-election-process/#voter-sign-up-and-eligibility[here].
+
+Once sign-up is over, the election committee will process the form submissions and prepare a list of the registered voters.
+In the case of rejection, one of the election committee members will send a rejection email.
+Every individual contributor is expected to vote only once.
+
+== Candidates
+
+Below you can find statements, affiliations and profile links provided by the candidates.
+//TODO: More text
```

Comment:
```suggestion
```
Is it really needed?

Reply:
I will probably add something as a follow-up. Should be good enough for the mailing list announcements today